packages/webpack-cli/lib/utils/cli-flags.js

```diff
@@ -286,11 +287,17 @@ module.exports = {
         {
             name: 'mode',
             usage: '--mode <development | production>',
-            type: String,
+            type: (value) => {
+                if (value === 'development' || value === 'production' || value === 'none') {
+                    return value
+                } else {
+                    logger.warn('You provided invalid value for "mode" option, webpack will fallback to "production" for this value.')
+                }
+            },
             group: ZERO_CONFIG_GROUP,
             description: 'Defines the mode to pass to webpack',
             link: 'https://webpack.js.org/concepts/#mode',
-            acceptedValues: ["development", "production"]
+            acceptedValues: ["development", "production", "none"]
```

Comment:
`acceptedValues` is not currently used. You can remove it

Reply:
@ematipico Done :+1: